### PR TITLE
Independent logit for covariate-informed and gene-expression-informed dimensions

### PR DESCRIPTION
**Pull request by lucas-diedrich**
This is related to pull requests #13 and #14 

The plate model assumes that sample-level and biological variability are independent from each other. However, the previous implementation enforces the logit normal constraint on the joint latent space of sample- and gene expression- informed dimensions. 

1) This leads to leakage of information between both modalities
2) During training, a (desired) high value in the sample embedding simultaneously enforces a lower value in all other dimensions (undesired separation of all dimensions due to separation in sample dimension). 

However, the logit normal distribution is beneficial for interpretability, as it enforces non-negativitiy (interpretation as program activity), and imposes a useful constraint on the latent space (reduced variability).  

The solution is to enforce non-negativity separately on the sample-informed dimensions via `torch.nn.Sigmoid()`  instead of joint logit normal.

### CODE REVIEW

**Review by lucas-diedrich — COMMENTED** [review]
Ok
